src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let cost = 0;
+
+  if (days > 0 && days < 3) {
+    cost = days * 40;
+  } else if (days >= 3 && days < 7) {
+    cost = days * 40 - 20;
+  } else {
+    cost = days * 40 - 50;
+  }
+
+  return cost;
```

Comment:
```suggestion
  if (days < 3) {
    return days * 40;
  }

  if (days < 7) {
    return days * 40 - 20;
  }

  return days * 40 - 50;
```

відʼємної кількості днів не може бути